schutzbot/deploy.sh

```diff
@@ -23,6 +19,8 @@ if [[ "$ID" == rhel ]] && sudo subscription-manager status; then
   DISTRO_VERSION=rhel-${VERSION_ID%.*}-cdn
 fi
 
+sudo dnf install -y jq
```

Comment:
Why was this added here? If it's needed, can we move it to the other dnf transaction?

Reply:
Hmm, randomness most probably :sweat_smile: . I'll merge that with the other.

Reply:
I'm taking care of that for the follow up, since I'm splitting this PR in two parts.